sdk/communication/communication-phone-numbers/samples-dev/updatePhoneNumberCapabilities.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Update the capabilities of a purchased phone number.
+ */
+
+import {
+  PhoneNumbersClient,
+  PhoneNumberCapabilitiesRequest
+} from "@azure/communication-phone-numbers";
+
+export const main = async () => {
+  console.log("\n== Update Phone Number Capabilities Typescript Sample ==\n");
+
+  // You will need to set this environment variable or edit the following values
+  const connectionString =
+    process.env.COMMUNICATION_CONNECTION_STRING ||
+    "endpoint=https://resourceName.communication.azure.net/;accessKey=test-key";
+
+  try {
+    // create new client
+    const client = new PhoneNumbersClient(connectionString);
+
+    // You will need to set this environment variable or edit the following values
+    const phoneNumberToUpdate = process.env.PHONE_NUMBER_TO_UPDATE || "";
```

Comment:
Should there be a place holder here instead of empty string? or throw if the environment variable doesnt exist?